backup/Dockerfile

```diff
@@ -8,7 +8,8 @@ RUN apt-get update && \
                 cron \
                 python3-pip \
                 python3-setuptools \
-                wget
+                wget \
+                python-flask
```

Comment:
can we [organise these alphabetically](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments) please?